java/api/src/main/java/org/ray/api/Ray.java

```diff
@@ -1,6 +1,9 @@
 package org.ray.api;
 
 import java.util.List;
+import org.ray.api.funcs.RayFunc_1_1;
+import org.ray.api.funcs.RayFunc_3_1;
+import org.ray.api.funcs.RayFunc_4_1;
```

Comment:
import org.ray.api.funcs.*;

Reply:
unfortunately, this is not supported in Java.